src/main/java/org/junitpioneer/jupiter/Shared.java

```diff
@@ -0,0 +1,11 @@
+package org.junitpioneer.jupiter;
+
+public @interface Shared {
+
+  Class<? extends ResourceFactory<?>> value();
```

Comment:
I think using `value` as a name for the attribute makes the most sense when you can expect that it will be the only configured attribute in most use cases (because you can spare the `value = `). But since we always need the `name` as well, users would always have to write `value`, in which case I think we should use a more explicit name. What about `factory`?

Reply:
Yep, that makes sense! It's now in the TODO list.